scripts/auto-rebase/rebase.sh

```diff
@@ -178,6 +178,16 @@ update_lvms_manifests() {
     for m in "${namespace_scoped_manifests[@]}"; do
         yq -i '.metadata.namespace = "openshift-storage"' "${dest}/${m}" || return 1
     done
+
+    # These manifests are not published and have to be gotten manually from the LVMS operator.
+    # See https://issues.redhat.com/browse/OCPVE-251 request to support MicroShift
```

Comment:
Take a look at https://github.com/openshift/microshift/blob/main/scripts/auto-rebase/assets.yaml#L60. We have a field `ignore: "reason"` and it documents why we don't touch some topolvm manifests during rebase.

Also we don't want to include jira urls in source code